pydatalab/Pipfile.lock

```diff
@@ -108,11 +108,11 @@
         },
         "anthropic": {
             "hashes": [
-                "sha256:419a276eb20cfb7ddaac03c7e28e4e12df3ace71bcf33071a68c9a03c0dfcbdd",
```

Comment:
What did you actually need to update for this PR to work? A minimum version should be added in the `Pipfile` too in that case (though I am also hoping #604 will sort this out sooner rather than later)